opennms-doc/guide-admin/src/asciidoc/text/telemetryd/protocols/jti/junos-telemetry-interface.adoc

```diff
@@ -6,7 +6,7 @@ _{opennms-product-name}_ sends a request to stream periodic updates once to the
 Data is generated as Google protocol buffers (gpb) structured messages over _UDP_.
 Detailed information about _JTI_ can be found in link:https://www.juniper.net/documentation/en_US/junos/topics/concept/junos-telemetry-interface-oveview.html[Juniper Documentation].
 
-To enable support for Junos Telemetry Interface (JTI), edit `${OPENNMS_HOME}/etc/telemetryd-configuration.xml` set `enabled=true`.
+To enable support for Junos Telemetry Interface (JTI), edit `${OPENNMS_HOME}/etc/telemetryd-configuration.xml` set `enabled=true` for `NXOS` protocol.
```

Comment:
Was the reference to `NXOS` here intentional?

Reply:
Nah, corrected.